### PR TITLE
Add close method to websocket client

### LINKED ISSUE #145: Interactive Exec on WebSocket api Close() requirement
Had a discussion with @mbohlool on slack about externalizing a close() method for websocket client. Scenario describe below:

**Scenario:**

1. I have opened up an interactive exec.
2. Executed a few commands.
3. Completed my job.

**Proposal:**

1. We need to externalize some method to close the socket in the event we are done using it and don't want the socket lingering around. Yes we could use an `exit` as part of our commands to run but does that really clean up the socket we opened up on our end?

### PR DESCRIPTION
**Pull request by mbohlool**
fixes #145

### CODE REVIEW

File: kubernetes/client/ws_client.py [hunk 1]
```diff
@@ -191,6 +191,14 @@ def run_forever(self, timeout=None):
             while self.is_open():
                 self.update(timeout=None)
 
+    def close(self, **kwargs):
+        """
+        close websocket connection.
+        """
+        self._connected = False
```
Comment (mrmcmuffinz) [inline kubernetes/client/ws_client.py]: Should we check that the socket is open before closing it?

File: kubernetes/client/ws_client.py [hunk 2]
```diff
@@ -191,6 +191,14 @@ def run_forever(self, timeout=None):
             while self.is_open():
                 self.update(timeout=None)
 
+    def close(self, **kwargs):
```
Comment (mrmcmuffinz) [inline kubernetes/client/ws_client.py]: What are some of the variable arguments you envision being used here? Just trying to get a feel for this change...

Comment (mrmcmuffinz) [inline kubernetes/client/ws_client.py]: Also, just out of curiosity, should we return True/False if we were able to close successfully the socket?

Comment (mbohlool) [inline kubernetes/client/ws_client.py]: Look at WebScoket close method for more information: https://github.com/websocket-client/websocket-client/blob/master/websocket/_core.py#L372
that method does not return any status.

File: kubernetes/client/ws_client.py (hunk 1 above)
Comment (mbohlool) [inline kubernetes/client/ws_client.py]: close method is safe to call even if the socket is closed.

File: kubernetes/client/ws_client.py (hunk 2 above)
Comment (mrmcmuffinz) [inline kubernetes/client/ws_client.py]: Got it, thanks.

File: kubernetes/client/ws_client.py (hunk 1 above)
Comment (mrmcmuffinz) [inline kubernetes/client/ws_client.py]: Got it, thanks.

**Pull request merged**